docs/challenges/language-support/python.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: "Python"
+description: "Learn how to solve CodeCrafters challenges in Python"
+---
+
+<Note>
+  **New to Python?** We recommend checking out [Automate the Boring Stuff with
+  Python](https://automatetheboringstuff.com/) to get up to speed.
+</Note>
+
+# Local development
+
+To run the code for your challenge locally, you'll need `Python` installed. Our test runners use `Python` version `3.12` (as of April 2024).
+
+The script for your challenge (like `./spawn_redis_server.sh`) will run the code using this version of the Python interpreter. It looks something like this:
+
+```sh
+#!/bin/sh
+
+exec python3 -m app.main "$@"
```

Comment:
Hmm, yeah it's a bit weird that this script differs - for a challenge that supports pipenv it is different, and for challenges that don't it is different. This is fine for now, we'll standardize these and then come back and update this.